cypress/e2e/widgets/widget-pip-close.spec.ts

```diff
@@ -0,0 +1,197 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
```

Comment:
this is not copyright to us - it's copyright to you.

Reply:
good point :) updated, thanks